[Tailwindを使用したことある方向け]Flexの理解を深めよう

w-shaulabg opened the pull request:
お疲れ様です！
[Tailwindを使用したことある方向け]Flexの理解を深めよう課題２から４を提出いたします！
お手隙の際にご確認のほどよろしくお願いいたします！

↓課題↓
<img width="641" alt="スクリーンショット 2022-11-23 20 35 29" src="https://user-images.githubusercontent.com/100771427/203537044-d99dd481-a68e-42e2-8afc-f83524b8a033.png">
<img width="1438" alt="スクリーンショット 2022-11-23 20 36 10" src="https://user-images.githubusercontent.com/100771427/203537171-f244d9fd-6405-432e-be08-2398dcafd55d.png">
<img width="743" alt="スクリーンショット 2022-11-23 20 36 43" src="https://user-images.githubusercontent.com/100771427/203537268-68acfa0d-c04b-47dc-8245-94b7dac697f4.png">

↓実行結果↓
課題２
<img width="495" alt="スクリーンショット 2022-11-23 20 40 42" src="https://user-images.githubusercontent.com/100771427/203537995-675ab7fc-6290-411d-9ec5-8540dde4b852.png">
課題３
<img width="1440" alt="スクリーンショット 2022-11-23 20 41 21" src="https://user-images.githubusercontent.com/100771427/203538139-de4d41e0-500d-4fb3-aa33-f22c10883d9a.png">
課題４
<img width="1440" alt="スクリーンショット 2022-11-23 20 42 19" src="https://user-images.githubusercontent.com/100771427/203538317-9e91e09f-1a2e-4232-a962-e843bd30eff9.png">

w-shaulabg committed:
[Tailwindを使用したことある方向け]Flexの理解を深めよう
課題２から４

gotokouichi reviewed (changes requested):
実装内容良い感じですね！😊

一点修正いただきたい点があります!✨
ディレクトリ構造についてです！👍
以下の参考サイトにもありますが、cssファイルはCSSディレクトリにまとめていただけると幸いです！
規模が大きくなってきた場合を想定すると、拡張子ごとにディレクトリ分けしたほうがより管理しやすくなるため、コメントさせていただきました！😊


https://pengi-n.co.jp/blog/folder-structure/
https://shuu1104.com/2021/04/2445/

w-shaulabg committed:
・ファイル移動
Tailwind1.htmlからTailwind4.html → TailwindHtml
tailwind2.css及びtailwind4.css　→ CSS

gotokouichi reviewed (approved):
LGTMです！✨
ご自身でマージしちゃってください！😊
